### DISCUSSION TITLE
Style Intervals depending on Category for v-calendar

### DISCUSSION THREAD

**Discussion by WobblySlug** [vuetifyjs/vuetify#13971] [Q&A] [answered]
Hey everyone. Is it possible to style the interval that the category differently? It appears that "interval" is a time slot across multiple categories.

Currently I have this (the grey area at the top indicates that it's outside of working hours)

![image](https://user-images.githubusercontent.com/5068726/126736920-b66be304-dcb3-4658-a56d-2e04067e3809.png)

But I'd like to be able to do this (style each interval category separately).

![image](https://user-images.githubusercontent.com/5068726/126736940-58aa655e-54b3-4c67-9046-7f2a2772b76c.png)

I'm open to any suggestions, thanks so much in advance.

**Comment by KaelWD** [comment]
Both interval-style and the interval slot expose the current category 
![image](https://user-images.githubusercontent.com/16421948/126961251-c307da14-1144-405f-8582-12dfb7de7153.png)

**Reply by WobblySlug** [reply]
Hey @KaelWD , thanks for your reply.

Could you please tell me which version of Vuetify are you using? The [API documentation I'm looking at here](https://vuetifyjs.com/en/api/v-calendar/#slots-interval) doesn't have anything like that.

Also, what is your console.log actually representing in your screenshot above?

Thanks in advance!

**Reply by KaelWD** [reply]
Looks like it's missing from the api docs. 

https://codepen.io/kaelwd/pen/xxdpWJp?editors=1010

**Reply by WobblySlug** [reply]
@KaelWD  dude I could kiss you!

Any idea what version of Vuetify has the category within the interval style? 
My package.json says `"vuetify": "^2.3.16"`, and unfortunately does not contain any category JSON property. I may have to think about updating soon!

**Reply by KaelWD** [reply]
2.4.0: #12518

**Reply by WobblySlug** [reply]
> 
> 
> 2.4.0: #12518

Thanks again @KaelWD

**Comment by WobblySlug** [comment, marked as answer]
The ability to style a category interval was added in Vuetify v2.4.0
